Source/ConfigCatWrappers/Private/ConfigCatTargetingRuleWrapper.cpp

```diff
@@ -0,0 +1,154 @@
+﻿// Copyright (c) ConfigCat 2024. All Rights Reserved.
+
+#include "ConfigCatTargetingRuleWrapper.h"
+
+#include "ConfigCatSettingValueContainerWrapper.h"
+#include "ConfigCatPercentageOptionWrapper.h"
+#include "ConfigCatValueWrapper.h"
+
+UConfigCatUserConditionWrapper* UConfigCatUserConditionWrapper::CreateUserCondition(const configcat::UserCondition& InUserCondition)
+{
+	UConfigCatUserConditionWrapper* Result = NewObject<UConfigCatUserConditionWrapper>();
+	Result->UserCondition = InUserCondition;
+	return Result;
+}
+
+FString UConfigCatUserConditionWrapper::GetComparisonAttribute() const
+{
+	return UTF8_TO_TCHAR(UserCondition.comparisonAttribute.c_str());
+}
+
+EConfigCatUserComparator UConfigCatUserConditionWrapper::GetComparator() const
+{
+	if (static_cast<int32>(UserCondition.comparator) < 0)
+	{
+		return EConfigCatUserComparator::Invalid;
+	}
+	
+	return static_cast<EConfigCatUserComparator>(UserCondition.comparator);
+}
+
+FString UConfigCatUserConditionWrapper::GetStringComparisonValue() const
+{
+	// UserCondition.comparisonValue;
```

Comment:
I think you can implement it like this: 
```suggestion
	const auto comparisonValuePtr = get_if<string>(&UserCondition.comparisonValue);
	if (comparisonValuePtr)
	{
		return UTF8_TO_TCHAR(comparisonValuePtr->c_str());
	}
```

Reply:
Thanks for spotting those and for the suggestions, I didn't know they were a thing.

However, I have to implement the `Has...Value` functions as well, so I will be using that to check the values.